migrations/migrations/20210331154903-users.js

```diff
@@ -2,39 +2,39 @@
 
 module.exports = {
   up: async (queryInterface, Sequelize) => {
-    await queryInterface.createTable(
-      'user',
-      {
-        id: {
-          type: Sequelize.INTEGER,
-          primaryKey: true,
-          autoIncrement: true,
-          allowNull: false,
-          unique: true
-        },
-        first_name: {
-          type: Sequelize.STRING,
-          allowNull: false
-        },
-        last_name: {
-          type: Sequelize.STRING,
-          allowNull: false
-        },
-        email: {
-          type: Sequelize.STRING,
-          allowNull: false,
-          unique: true
-        },
-        password: {
-          type: Sequelize.STRING,
-          allowNull: false
-        }
+    await queryInterface.createTable('users', {
+      id: {
+        type: Sequelize.INTEGER,
+        primaryKey: true,
+        autoIncrement: true,
+        allowNull: false,
+        unique: true
       },
-      {
-        freezeTableName: true
+      first_name: {
+        type: Sequelize.STRING,
+        allowNull: false
+      },
+      last_name: {
+        type: Sequelize.STRING,
+        allowNull: false
+      },
+      email: {
+        type: Sequelize.STRING,
+        allowNull: false,
+        unique: true
+      },
+      password: {
+        type: Sequelize.STRING,
+        allowNull: false
+      },
+      createdAt: {
```

Comment:
Remember table properties should be on snake_case

Reply:
But the createdAt and updatedAt properties are named in camelCase inside the Database